plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/preferences/SSLCertificatesPreferencePage.java

```diff
@@ -0,0 +1,201 @@
+/*******************************************************************************
+ * Copyright (c) 2015 Red Hat, Inc.
+ * Distributed under license by Red Hat, Inc. All rights reserved.
+ * This program is made available under the terms of the
+ * Eclipse Public License v1.0 which accompanies this distribution,
+ * and is available at http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     Red Hat, Inc. - initial API and implementation
+ ******************************************************************************/
+package org.jboss.tools.openshift.internal.ui.preferences;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.eclipse.jface.preference.FieldEditorPreferencePage;
+import org.eclipse.jface.viewers.CheckboxTableViewer;
+import org.eclipse.jface.viewers.ISelectionChangedListener;
+import org.eclipse.jface.viewers.IStructuredContentProvider;
+import org.eclipse.jface.viewers.IStructuredSelection;
+import org.eclipse.jface.viewers.ITableLabelProvider;
+import org.eclipse.jface.viewers.LabelProvider;
+import org.eclipse.jface.viewers.SelectionChangedEvent;
+import org.eclipse.jface.viewers.Viewer;
+import org.eclipse.swt.SWT;
+import org.eclipse.swt.events.SelectionEvent;
+import org.eclipse.swt.events.SelectionListener;
+import org.eclipse.swt.graphics.Image;
+import org.eclipse.swt.layout.GridData;
+import org.eclipse.swt.layout.GridLayout;
+import org.eclipse.swt.widgets.Button;
+import org.eclipse.swt.widgets.Composite;
+import org.eclipse.swt.widgets.Label;
+import org.eclipse.ui.IWorkbench;
+import org.eclipse.ui.IWorkbenchPreferencePage;
+import org.jboss.tools.openshift.internal.ui.preferences.SSLCetrificatesPreference.Item;
+
+public class SSLCertificatesPreferencePage extends FieldEditorPreferencePage implements IWorkbenchPreferencePage {
+	List<Item> items = new ArrayList<>();
+
+	CheckboxTableViewer listViewer;
+	Button remove;
+	
+	
+	public SSLCertificatesPreferencePage() {
+		super(GRID);
+	}
+
+	@Override
+	public void init(IWorkbench workbench) {
+		setPreferenceStore(SSLCetrificatesPreference.getInstance().getPreferenceStore());		
+		items.addAll(SSLCetrificatesPreference.getInstance().getSavedItems());
+	}
+
+	@Override
+	protected void createFieldEditors() {
+		Composite composite = getFieldEditorParent();
+		adjustGridLayout();
+
+		Label label = new Label(composite, SWT.NONE);
+		GridData dl = new GridData();
+		dl.horizontalSpan = 3;
+		label.setLayoutData(dl);
+		label.setText("Decisions on untrusted SSL certificates, checked ones are accepted.");		
+		
+		listViewer = CheckboxTableViewer.newCheckList(composite, SWT.BORDER | SWT.V_SCROLL);
+		listViewer.setColumnProperties(new String[]{"Certificate"});
+        GridData data = new GridData(GridData.FILL_HORIZONTAL);
```

Comment:
The formatting is all over the place. Would be nice if it was consistent on the 1st commit :-)